### ISSUE TITLE
XML File

### ISSUE THREAD

**Issue by versavius** [alexmbird/uk-it-recruiter-domains#112]
Is it possible to get a pre-generated xml file uploaded for us to download?

Happy to keep contributed, I have added 300 domains to be added to your list.

**Comment by alexmbird** [comment]
The model I'm following is to have a single, canonical list of domains in plain text and scripts you can run locally to present the data in other formats.  A script to output XML should be easy enough - can you give an example of the format you'd expect to find records in?

**Comment by versavius** [comment]
Oh I think i misunderstood. I thought you were preparing an XML list for gmail which is what I use.

I didn't know if it was possible to have an uploaded list, its not a problem if its too tricky, i can go through it and update it offline.

**Comment by alexmbird** [comment]
You're right!  Turns out we already have a [Ruby script](https://github.com/alexmbird/uk-it-recruiter-domains/blob/master/scripts/domains2gmail.rb) to output gmail xml.

I hadn't considered this before but it would be possible to setup a "build" process in GitHub actions to compile the list into various formats then publish it.  I wouldn't want them in the repo, they'd be more like a release artifact from a conventional project.

**Comment by versavius** [comment]
That would be perfect, I think if you can get github to generate the xml file maybe more people will contribute :) and use your lists :) I  know i'll be adding to your list again in the future :)
